docs/worker/monitoring.rst

```diff
@@ -0,0 +1,13 @@
+Monitoring the Celery Worker
```

Comment:
This page is describing how to set `log_url` for runs. This is not really a worker monitoring (e.g. Prometheus integration)